src/components/CytoscapeGraph/EmptyGraphLayout.tsx

```diff
@@ -25,6 +26,19 @@ export default class EmptyGraphLayout extends React.Component<EmptyGraphLayoutPr
     MessageCenter.toggleMessageCenter();
   };
 
+  shouldComponentUpdate(nextProps: EmptyGraphLayoutProps) {
+    // Update if we are going from having no elements to having elements or vice versa
+    if (_.isEmpty(this.props.elements) !== _.isEmpty(nextProps.elements)) {
```

Comment:
are we using "lodash" as a standard, or just as a helper for these scenarios where we don't have types ?
I see on this case is necessary as any could be any JSX element, but I have seen it in a couple of places and I would like to know some details, in order to consolidate the best-practice about when to use it or not.

Reply:
I'm using as a last resort, I don't recall to be using `lodash` anywhere else.
Currently the elements can be "{}" or the full graph object.